test/cppsuite/test_harness/util/api_const.h

```diff
@@ -47,6 +47,7 @@ extern const std::string CACHE_SIZE_MB;
 extern const std::string CC_PAGES_REMOVED;
 extern const std::string COLLECTION_COUNT;
 extern const std::string COMPRESSION_ENABLED;
+extern const std::string CUSTOM_CONFIG;
```

Comment:
I still feel like this is a bit confusing, I know its not consistent but we may want to think of making that config "custom_op_config" or something.

Reply:
I have renamed the different constants to make it clearer, see b0d25586d4a017f10a2ab15c2282c3d5c35e6760